ee/clickhouse/queries/funnels/funnel_event_query.py

```diff
@@ -25,6 +25,11 @@ def get_query(self, entities=None, entity_name="events", skip_entity_filter=Fals
             for column_name in self._column_optimizer.event_columns_to_query
         )
 
+        _fields.extend(
```

Comment:
Aside: This is gonna make materializing group properties annoying isn't it? :D

Reply:
Definitely. We're also going to have to "push querying down" to the GroupsJoinQuery, because otherwise the joins don't get the materialised columns.

And then adding those here, and seeing we aren't missing any so we don't have to query `group_properties_X`. Yeah, annoying.